packages/account/package.json

```diff
@@ -49,7 +49,8 @@
         "react-dom": "^17.0.2",
         "react-i18next": "^11.11.0",
         "react-router": "^5.2.0",
-        "react-router-dom": "^5.2.0"
+        "react-router-dom": "^5.2.0",
+        "yup": "^0.32.11"
```

Comment:
How come we're using `yup`. Isn't `zod` preferred here?

Reply:
We're trying new validation method in this file by using `Yup` package.

Reply:
@markwylde-deriv, I think we have chosen Yup because Bot team already uses it, and just because Formik documentation refers to Yup as recommended :)
if we could move to React Hook Form and Zod - this would be great :)

Reply:
> React Hook Form and Zod 

Let's do it sometime 🥇